api/handlers.go

```diff
@@ -658,6 +658,9 @@ func (si *ServerImplementation) fetchAccounts(ctx context.Context, options idb.A
 			account = row.Account
 		}
 
+		// Adjust rewards field to match algod
```

Comment:
I'd add a word about the "adjustment" to market it a bit more clear, but looks good.